lambda/index.js

```diff
@@ -1 +1,59 @@
-// TODO Implement lambda
\ No newline at end of file
+const zlib = require('zlib')
+const https = require('https')
+
+function sendLogEventsToHumio (logEvents) {
+  const data = JSON.stringify([{
```

Comment:
I agree. I just can't think of another way to do it. We have to parse it to get the `timestamp` and `message` from the Lambda separately, but then have to stringify it to pass it with the `https` library. Is there some other way I could approach it?

Reply:
The main goal was to try to avoid `JSON.stringify(JSON.parse(event.message))`, especially since those messages might end up being large nested objects.

It looks like you already took out the `JSON.parse(event.message)`, so the rest of this doesn't really apply anymore, but just to finish up the conversation:

The alternatives are template strings, like I alluded to
```js
const data = `[{"tags":{"SubIdxNM":"${process.env.SUB_IDX_NM}"},"events":[${logEvents.map(event=>`{"timestamp":"${event.timestamp}","timezone":"America/Denver","attributes":${event.message}}`).join(',')}]}]`
```
or using libraries like [`fast-json-stringify`](https://github.com/fastify/fast-json-stringify) that get to take shortcuts when stringifying because you hand them a schema up front.

That template string isn't very readable, though. And with the lack of any demonstrated performance issues, I'd err on the side of readability.